### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ani-exists**
- Added script for pin-cpu
- Added script for llm-demo

### CODE REVIEW

File: llm-demo/infer_server.py [hunk 1]
```diff
@@ -0,0 +1,802 @@
...
+*  Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+*  Neither the name of Intel Corporation nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR APARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHTOWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOTLIMITEDTO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANYTHEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF -THE USEOF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+"""
+
+import os
+
+os.getcwd()
```
Comment (s-pankratov) [inline llm-demo/infer_server.py]: return value is not used anywhere, should be removed

File: llm-demo/infer_server.py [hunk 2]
```diff
@@ -0,0 +1,802 @@
...
+    TextStreamer,
+)
+from termcolor import colored
+from peft import PeftModel
+from threading import Thread
+import intel_extension_for_pytorch as ipex
+
+Tokenizer = Union[PreTrainedTokenizer, PreTrainedTokenizerFast]
+
+import os
```
Comment (s-pankratov) [inline llm-demo/infer_server.py]: Some double imports, like `os` and  `time`

File: llm-demo/infer_server.py (hunk 1 above)
Comment (s-pankratov) [reply llm-demo/infer_server.py]: Also seems to be there is no usage for `os`, except stray getcwd

File: llm-demo/infer_server.py [hunk 3]
```diff
@@ -0,0 +1,802 @@
...
+
+details = """
+## Software Configuration Details:
+<br>
+To learn more about the Intel software and tools used in this demo, visit developer.intel.com/ai
+"""
+
+hw = """
+## Hardware Details:
+Intel® Xeon® Platinum 8488C @2.4GHz, Cores per Socket: 4, Threads per Core: 2. Running on AWS m7i.2xlarge EC2 instance.
```
Comment (s-pankratov) [inline llm-demo/infer_server.py]: Probably we shouldn't have this text there at all, or add functionality to report this properly. This text doesn't seems to be used anywhere (at least explicitly).

Comment (s-pankratov) [reply llm-demo/infer_server.py]: Check other text fragments - do we really need them, or can drop

File: llm-demo/infer_server.py (hunk 2 above)
Comment (ani-exists) [reply llm-demo/infer_server.py]: Removed the copies

File: llm-demo/infer_server.py (hunk 1 above)
Comment (ani-exists) [reply llm-demo/infer_server.py]: Removed the import

File: llm-demo/infer_server.py (hunk 3 above)
Comment (ani-exists) [reply llm-demo/infer_server.py]: Dropped the unnecessary ones. Only left title in since we use that

**Review by s-pankratov — APPROVED** [review]
LGTM

**Pull request merged**